api/envoy/config/wasm/v2alpha/wasm.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
```

Comment:
So, is there also an HTTP filter Wasm extension that then embeds this message? I'm curious how this wires into the existing extension points?

Reply:
yes, you can see that at https://github.com/envoyproxy/envoy-wasm/blob/master/api/envoy/config/filter/http/wasm/v2/wasm.proto